taxi/tests/test_models.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+class ModelTestCase(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="tests",
+                                                   country="testcountry")
```

Comment:
fix formatting also here, please, to make it consistent (like you formatted same line in the next test)